easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -152,7 +166,8 @@ function prepend_miniconda(){
         echo "Miniconda installed."
 
         echo "Prepending miniconda to PATH..."
-        export PATH="$MINICONDA_PATH/bin:$PATH"
+        PATH="$MINICONDA_PATH/bin:$PATH"
```

Comment:
Why remove the export?

Reply:
Isn't `PATH` always declared for export?